docs/release-notes.txt

```diff
@@ -3,6 +3,134 @@
 Release Notes
 =============
 
+Version 0.13
+------------
+
+Version 0.13.0
+^^^^^^^^^^^^^^
+
+**Release Date:** August 20, 2020
+
+**New Features**
+
+- Allow experiments to connect to the master using TLS.
```

Comment:
"Experiments" aren't really the things connecting to the master -- "trials"? "Training workloads"?